code/index.html

```diff
@@ -3,28 +3,224 @@
+          </div>
+          <div class="faq-content" id="faqContentFour">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Deleniti tempora nostrum, in nesciunt, iusto atque mollitia, sed libero odio ut iure. Explicabo dolore suscipit ipsum tempora. Voluptates iste in repellendus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet, impedit totam? Corrupti deserunt, ipsa, dolorem quas rem nemo dignissimos quam, ea modi rerum laboriosam veniam repellat nesciunt molestiae dicta at. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+
+        <div class="faq-container">
+          <div class="faq-header" id="faqHeaderFive">
+            <h1>Header five</h1> <i id="fasHeaderFive" class="fas fa-angle-down"></i>
+          </div>
+          <div class="faq-content" id="faqContentFive">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Deleniti tempora nostrum, in nesciunt, iusto atque mollitia, sed libero odio ut iure. Explicabo dolore suscipit ipsum tempora. Voluptates iste in repellendus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet, impedit totam? Corrupti deserunt, ipsa, dolorem quas rem nemo dignissimos quam, ea modi rerum laboriosam veniam repellat nesciunt molestiae dicta at. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+
+        <div class="faq-container">
+          <div class="faq-header" id="faqHeaderSix">
+            <h1>Header six</h1> <i id="fasHeaderSix" class="fas fa-angle-down"></i>
+          </div>
+          <div class="faq-content" id="faqContentSix">
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Deleniti tempora nostrum, in nesciunt, iusto atque mollitia, sed libero odio ut iure. Explicabo dolore suscipit ipsum tempora. Voluptates iste in repellendus. Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet, impedit totam? Corrupti deserunt, ipsa, dolorem quas rem nemo dignissimos quam, ea modi rerum laboriosam veniam repellat nesciunt molestiae dicta at. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
       </div>
+
     </section>
-    <!-- dont forget to link the javascript file here! -->
+
+
+    <!-- SIGN UP -->
+    <section class="sign-up-section" id="sign-up-section">
+      
+      <div class="sign-up-box">
+        <h2 class="sign-up-h2">
+          sign up
+        </h2>
+
+        <form class="sign-up-form" id="sign-up-form" action="http://httpbin.org/anything" method="POST">
+        
+          <label for="name">Name</label></br>
+          <div class="sign-up-control">
+            <input class="input-text" type="text" placeholder="John Doe" name="name" required>
+            <div class="form-required name">
+              <i class="fas fa-exclamation"></i>
+              <small>Please fill in your name</small>
+            </div>
```

Comment:
Hmm, I don't this you need this code, at least I don't see any place it will be used.
Moreover, you need to add `id` to inputs so they are connected to labels
You should rather aim for this schema :

```
<label for="name">Name</label>
<input id="name" class="input-text" type="text" placeholder="John Doe" name="name" required>
```

Let's fix this